File: KOTLIN/1011.kt

import java.util.Scanner
import java.lang.Math

fun main(args:Array<String>){
  val sc = Scanner(System.`in`) 
  val r: Double = sc.nextDouble()
  println(saida(volume(r)))

  
}
fun volume(r:Double):Double = Math.pow(r,3.0)* (4.0/3) * 3.14159
fun saida(vlr:Double ):String = "VOLUME = %.3f".format(vlr)

File: GO/1011.go
package main

import (
	"fmt"
	"math"
)

const PI float64 = 3.14159

func Volume(raio int64) float64 {
	return (4.0 / 3.0) * PI * math.Pow(float64(raio), 3)
}
func main() {
	var raio int64
	fmt.Scan(&raio)

	fmt.Printf("VOLUME = %.3f\n", Volume(raio))
}
